src/m365/spo/commands/page/page-publish.ts

```diff
@@ -0,0 +1,85 @@
+import { Logger } from '../../../../cli/Logger.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import { formatting } from '../../../../utils/formatting.js';
+import { urlUtil } from '../../../../utils/urlUtil.js';
+import { validation } from '../../../../utils/validation.js';
+import SpoCommand from '../../../base/SpoCommand.js';
+import commands from '../../commands.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  name: string;
+}
```

Comment:
> Notice that we require all new commands to use the ZOD scheme instead of options interfaces. This is our new way of working. All other commands should be refactored over time. This time we'll tolerate it, but keep in mind that all news commands should use a ZOD scheme.

I have found a command where ZOD has been used. I will make a similar change.

Reply:
I have found a command where ZOD has been used. I will make a similar change.

Reply:
Thx a lot!